history/characters/53000_hillsbrad.txt

```diff
@@ -446,4 +446,22 @@
 		}
 	}
 	620.3.13={ death=yes }
+}
+
+#Anetta Crank
+53100={
```

Comment:
Her link leads to a not-existing page.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81499569-9e54b280-92dd-11ea-8246-260dee30d78a.png)

</details>